scripts/pre-week.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+export NVM_DIR="/root/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh"  # This loads nvm
+
+# TODO: It might be nice to get up to date on node version.
+nvm use 6
+
+LOG=/home/mnp/main/data/cronlog/pre-week.log
+
+echo "Running Pre week:" `date` >> $LOG
+
+cd /home/mnp/data-archive
+git checkout master
+git pull origin master
+
+# TODO Find a way to make the script not season hard coded.
```

Comment:
Without a proper data system (or at least a better one), it may become difficult to do much across seasons. We have a model in the `server` project, which has most of the foundation of what you need.